assets/css/app.css

```diff
@@ -66,10 +66,37 @@ body {
 }
 
 .form-section {
+    display: flex;
+    flex-direction: column;
     margin: 0 2rem 0 2rem;
 }
 
-.form-text {
+.form-section-half {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+}
+
+.half-right {
+    margin: 0 0 0 1rem;
```

Comment:
isn't this just margin-left: 1rem;?

Reply:
Resolved with 92244839629182a73fc9096c4664cc861b65b1a2